nuntium/user_section/forms.py

```diff
@@ -41,6 +42,18 @@
 
 
 class WriteItInstanceBasicForm(ModelForm):
+    language = ChoiceField(choices=settings.LANGUAGES)
+
+    def __init__(self, *args, **kwargs):
+        kwargs['initial']['language'] = kwargs['instance'].config.default_language
```

Comment:
If this form is ever used for creation, not just editing, instance won't exist. Could just check it exists before setting this line?